### PR TITLE
Feature/add member to activity

### PR DESCRIPTION
**Pull request by Rusty89**

### CODE REVIEW

**Review by QuaziBit — APPROVED** [review]
Looks good

File: client-app/src/app/activity-involvement/activity-inv-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    }
+	
+	
+	private viewMembers(involvement: Involvement){
+		this.involvementsByName = [];
+		if((this.involvement.memberIDs!=null) && (this.members !=null)){
+			for (var i = 0; i < this.involvement.memberIDs.length; i++) {
+                for (var j = 0; j < this.members.length; j++) {
+                    if (this.involvement.memberIDs[i] === this.members[j].id) {
+                        this.involvementsByName.push(this.members[j].firstName.concat(" ".concat(this.members[j].lastName)))
```
Comment (RialJohnson) [inline client-app/src/app/activity-involvement/activity-inv-form.component.ts:96]: Curious as to why you chose to use .concat here. Was it necessary to make something work or was it just a best practice thing?

**Review by RialJohnson — APPROVED** [review]
Great team effort to add this functionality. Took a lot of coding and troubleshooting on both the front and back end and now functions how we want. <3

**Comment by RialJohnson** [comment]
> _No description provided._

Would be a good practice to document the changes you made when making a pull request.

File: client-app/src/app/activity-involvement/activity-inv-form.component.ts (hunk 1 above)
Comment (Rusty89) [reply client-app/src/app/activity-involvement/activity-inv-form.component.ts:96]: Just using it to put them together, was having trouble getting them to add up with +, it was doing unexpected stuff.

**Pull request merged**